.github/workflows/pull-request.yml

```
@@ -449,3 +452,144 @@
+            --categorizeByGroup true \
+            --entryPointStrategy expand \
+            --entryPoints "src/**/*.ts" "src/**/*.tsx" \
+            --exclude "**/*.{test,spec,stories}.{ts,tsx}" \
+            --exclude "**/__tests__/**" \
+            --exclude "**/__mocks__/**" \
+            --exclude "**/node_modules/**" \
+            --cleanOutputDir true
+
+      # Step 5: Create placeholder documentation (only if not present)
+      - name: Create Placeholder Documentation
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2028:info:5:8: echo may not expand escape sequences. Use printf

(shellcheck)